src/base/default_allocator.rs

```diff
@@ -46,7 +46,7 @@ where
 
     #[inline]
     unsafe fn allocate_uninitialized(_: R, _: C) -> Self::Buffer {
-        mem::uninitialized()
+        mem::MaybeUninit::uninit().assume_init()
```

Comment:
This is still undefined behavior, but at least the function is `unsafe`. This is the best we can do with the existing public API. See https://github.com/rustsim/nalgebra/issues/556 for more info on how this can be fixed.

Reply:
Yea I agree... Unfortunately I no longer have much time on this now...